src/main/steps/c100-rebuild/child-details/navigationController.test.ts

```diff
@@ -4,6 +4,8 @@ import {
   C100_CHILDERN_DETAILS_CHILD_MATTERS,
   C100_CHILDERN_DETAILS_PARENTIAL_RESPONSIBILITY,
   C100_CHILDERN_DETAILS_PERSONAL_DETAILS,
+  C100_CHILDERN_OTHER_CHILDREN_NAMES,
```

Comment:
Have a separate navigation controller test case for other children.

Reply:
Understood